staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder.go

```diff
@@ -172,8 +187,30 @@ func BuildSwagger(crd *apiextensionsv1.CustomResourceDefinition, version string,
 	for _, route := range routes {
 		b.ws.Route(route)
 	}
+	return b, nil
+}
 
-	openAPISpec, err := openapibuilder.BuildOpenAPISpec([]*restful.WebService{b.ws}, b.getOpenAPIConfig())
+func BuildOpenAPIV3(crd *apiextensionsv1.CustomResourceDefinition, version string, opts Options) (*spec3.OpenAPI, error) {
+	b, err := generateBuilder(crd, version, opts)
+	if err != nil {
+		return nil, err
+	}
+
+	openAPIv3Spec, err := builder3.BuildOpenAPISpec([]*restful.WebService{b.ws}, b.getOpenAPIConfig(false))
+	if err != nil {
+		return nil, err
+	}
+	return openAPIv3Spec, err
+}
+
+// BuildSwagger builds swagger for the given crd in the given version
+func BuildSwagger(crd *apiextensionsv1.CustomResourceDefinition, version string, opts Options) (*spec.Swagger, error) {
```

Comment:
BuildOpenAPIV2

I think we have mostly banned the term Swagger from the code by now. If not, we should.